Docs 9549 - Updates for editorial review (#106)
mulesoft/docs-partner-manager: modules/ROOT/pages/configure-message-flows.adoc

> = Create and Configure an Inbound Message Flow

Reviewer: This is more clearly Create and Configure than the similar topic in PR105

> Create an inbound message flow to receive EDI messages from your trading partners, validate, transform, and then send the translated application (JSON/XML) message to your back-end applications.

Reviewer: ```suggestion
Create an inbound message flow to receive EDI messages from your trading partners, validate and transform that message, and then send the translated application (JSON/XML) message to your back-end applications.
```
...or some other wording that makes this 3 steps: receive, process, and forward/send.

. Select an existing Partner from the list and click *Select*. +
A new message flow is created. +

> Alternatively, from the *Select a partner* dialog, you can click *New* to create a new partner.

Reviewer: These are two separate actions and I think this needs to be clearer; either select an existing partner (lowercase) or create a new partner.

Author: Done

> Follow the process to xref:configure-partner.adoc#create-a-trading-partner[create a Trading Partner], except, when you are finished, click *Create Partner and Message Flow* instead of *Create partner*.

Reviewer: Wait...this is really confusing. The task is supposed to be creating a flow, not creating a partner.
And the first sentence about configuring your payload storage API should be a prereq with a link to THAT task.
Instead of ref'ing the partner creating task, I would specify all the steps to specifically create a flow, even if it is repetition.

Author: Fixed!

== Configure the Receive Endpoint

. In the message flow, expand the *Receiving endpoint* section and click *Select*.
. To use an existing receiving endpoint:
.. Select the endpoint from the list and click *Select* in the lower right side of the page. +

> The endpoint is added to the message flow, and you see a message that tells you the endpoint is missing an identifier.

Reviewer: Should this treatment be applied in the previous task as well, re. existing vs. new?
Make the parallel tasks consistent.

Author: These are a bit different - the process is not required in the previous task, so I am leaving it alone.

== Configure the Source Message Type

> . In the messasge flow, expand the *Source message type* section, and click *Select*.

Reviewer: no comma
```suggestion
. In the messasge flow, expand the *Source message type* section and click *Select*.
```

> . To use an existing message type:

Reviewer: ditto

Author: Parallel to previous section, but not first since it does not apply to the first.

== Import the Translation Map

> In the message flow, expand the *Map* section and click *Import* to upload the DataWeave map to translate the EDI message to your application message format.

Reviewer: Not sure that this one step qualifies as its own task.

Author: Done

> Follow xref:partner-manager-maps.adoc[these instructions] to create your inbound EDI-to-application message (JSON or XML) map in Anypoint Studio.

Reviewer: This is a separate task...in fact, one that precedes translation, right?

Author: Done

== Configure the Target Message Type

. In the message flow, expand the *Target at <your organization>* section.

> . Click *Select* to select the message type for your backend system to map the EDI transaction. +

Reviewer: ```suggestion
. Select a message type for your backend system to use to map the EDI transaction. +
```
or
```suggestion
. Click *Select* and select a message type for your backend system to use to map the EDI transaction. +
```

Reviewer: ...but I am still confused. How is selecting a message type really configuring a message type, and shouldn't this really be part of a larger configuration task?

Partner Manager routes inbound EDI transactions based on matching identifiers configured at the Partner level.

== Configure the Target Endpoint

. In the message flow, select the endpoint to which to send the translated message (JSON or XML) for further processing in your backend system. +

> In the *Select endpoint* window, click *New* to create a new endpoint. +

Reviewer: Is this a step?
Again, this seems not really configuring the endpoint but rather just choosing the endpoint as part of a larger configuration task.

Author: Based on the UI, this is actually one section, so these steps have been combined.
